Source/WebKit/UIProcess/WebProcessProxy.cpp

```diff
@@ -3019,7 +3019,7 @@ void WebProcessProxy::registerServiceWorkerClients(CompletionHandler<void()>&& c
 HashMap<WebCore::PageIdentifier, CoreIPCAuditToken> WebProcessProxy::presentingApplicationAuditTokens() const
 {
     HashMap<WebCore::PageIdentifier, CoreIPCAuditToken> presentingApplicationAuditTokens;
-    WTF::forEach(pages(), [&] (auto& page) {
+    std::ranges::for_each(pages(), [&] (auto& page) {
```

Comment:
Its not clear to me this is more clear than:

```c++
for (auto& page : pages()) {
    if (page->presentingApplicationAuditToken())
        presentingApplicationAuditTokens.add(page->webPageIDInMainFrameProcess(), page->presentingApplicationAuditToken().value());
}
```

Generally, I think we should reserve use of the for_each() algorithm to when it is needed in composition with other range/view algorithms and use standard range-for in cases like these.